config/initializers/webmock.rb

```diff
@@ -0,0 +1,14 @@
+def mock_workflow_requests
+  escaped_url = Settings.WORKFLOW_URL.gsub('/', '\/')
+  stub_request(:get, /#{escaped_url}workflow_archive.*/)
+    .to_return(body: '<objects count="1"/>')
+  stub_request(:get, /#{escaped_url}dor.*/)
+    .to_return(body: '<workflows/>')
```

Comment:
i guess i agree with hound, since i don't mind long lines, but this reads fine, and i'm neutral on changing it.